src/api/payment-requests/payment-requests.md

```diff
@@ -467,12 +467,168 @@ them to find the Payment Request and proceed to pay.
 {% endjson %}
 
 
+### Pay a Payment Request by Id
```

Comment:
Do we want to mark these as experimental as there are some unresolved Qs around multi-asset refunds etc?

Reply:
Yes I think so. Overall this looks nice and simple. There's a bunch of context you might be missing so I've added notes to the internal design.

Reply:
Yeah, agreed. We will add the experimental flag to it.